src/action.js

```diff
@@ -33,8 +33,60 @@ function log(message) {
   console.log(`[${new Date().toUTCString()}] ${message}`);
 }
 
+function pad(n) {
+    var s = '000' + n;
+    return s.substring(s.length-4);
+}
+
 function normalizeName(problemName) {
-  return problemName.toLowerCase().replace(/\s/g, '_');
+  return problemName.toLowerCase().replace(/\s/g, '-');
+}
+
+async function getInfo(submission, session, csrf) {
+  let data = JSON.stringify({
+    query: `query submissionDetails($submissionId: Int!) {
+      submissionDetails(submissionId: $submissionId) {
+        runtimePercentile
+        memoryPercentile
+        question {
+          questionId
+        }
+      }
+    }`,
+    variables: {'submissionId':submission.id}
+  });
+
+  let config = {
+    maxBodyLength: Infinity,
+    headers: {
+      'Content-Type': 'application/json',
+      'Cookie': `LEETCODE_SESSION=${session};csrftoken=${csrf};`,
+    },
+    data : data
+  };
+
+  // No need to break on first request error since that would be done when getting submissions
+  const getInfo = async (maxRetries = 5, retryCount = 0) => {
+    try {
+      const response = await axios.post('https://leetcode.com/graphql/', data, config);
+      const runtimePercentile = `${response.data.data.submissionDetails.runtimePercentile.toFixed(2)}%`;
+      const memoryPercentile = `${response.data.data.submissionDetails.memoryPercentile.toFixed(2)}%`;
+      const questionId = pad(response.data.data.submissionDetails.question.questionId.toString());
+
+      log(`Got info for submission #${submission.id}`);
+      return { runtimeperc: runtimePercentile, memoryperc: memoryPercentile, qid: questionId };
```

Comment:
nit: Could we do `runtimePerc` and `memoryPerc` for consistency with other naming?